script/foo-bar.json

```diff
@@ -0,0 +1,81 @@
+{
+  "consumer": {
+    "name": "Foo"
+  },
+  "provider": {
+    "name": "Bar"
+  },
+  "interactions": [
+    {
+      "description": "a request to list the latest pacts",
+      "providerState": "a pact between Condor and the Pricing Service exists",
+      "request": {
+        "method": "get",
+        "path": "/pacts/latest",
+        "headers": {
+        }
+      },
+      "response": {
+        "status": 200,
+        "headers": {
+          "Content-Type": "application/hal+json"
+        },
+        "body": {
+          "_links": {
+            "self": {
+              "href": "http://example.org/pacts/latest"
+            }
+          },
+          "pacts": [
+            {
+              "_links": {
+                "self": [
+                  {
+                    "href": "http://example.org/pacts/provider/Pricing%20Service/consumer/Condor/latest"
```

Comment:
Good Ol' Condor and the Pricing Service

Reply:
They'll always live on in our test data.